src/lwc/utilInput/utilInput.js

```diff
@@ -196,13 +214,75 @@ export default class utilInput extends LightningElement {
             this.value = event.target.value;
         }
 
+        let isValid = true;
+        if (this.isLightningRichText) {
```

Comment:
I'm wondering if we need this condition - the default check (this.isValid()) calls checkFieldValidity() and if the type is rich text, that calls this.checkRichTextValidity() ... am I right in thinking that isValid() checks rich text validity as well, and that you can just reduce this block to: 

isValid = this.isValid()

Reply:
That's totally right! Updated!